src/coord_finder.cpp

```diff
@@ -74,15 +89,15 @@ class CoordFinder {
 
         // check if the PropInProgress message is valid
         if (prop_msg_.prop_label.empty()) {
-            ROS_WARN("Invalid PropInProgress message received - Prop type is empty");
+            ROS_WARN_STREAM(TAG << "Invalid PropInProgress message received - Prop type is empty");
             return;
         }
         if (std::isnan(prop_msg_.theta_1)) {
-            ROS_WARN("Invalid PropInProgress message received - theta 1 is empty");
+            ROS_WARN_STREAM(TAG << "Invalid PropInProgress message received - theta 1 is empty");
             return;
         }
         if (std::isnan(prop_msg_.theta_2)) {
-            ROS_WARN("Invalid PropInProgress message received - theta 2 is empty");
+            ROS_WARN_STREAM(TAG << "Invalid PropInProgress message received - theta 2 is empty");
             return;
         }
```

Comment:
I'm wondering if these should return. If theta_1 is empty and theta_2 is empty we probably want to know both of them are empty in the logs.

Reply:
If these warnings get called there's no point in continuing? We need all that information. We can set any of those messages to go to logging, just not set up yet.

Reply:
Offline chat: remove return statements so all errors will be logged